fendermint/vm/actor_interface/src/ipc.rs

```diff
@@ -197,6 +199,47 @@ pub fn subnet_id_to_eth(subnet_id: &SubnetID) -> Result<(u64, Vec<et::Address>),
     Ok((subnet_id.root_id(), route))
 }
 
+/// Hash some value in the same way we'd hash it in Solidity.
+///
+/// Be careful that if we have to hash a single struct,
+/// Solidity's `abi.encode` function will treat it as a tuple,
+/// so it has to be passed as a tuple in Rust. Vectors are fine.
+pub fn abi_hash<T: Tokenize>(value: T) -> [u8; 32] {
```

Comment:
@cryptoAtwill, we may need to use this one in the relayer when we check the hash of the messages for the checkpoint. It may make our lives significantly easier :)

Reply:
I didn't think you have to check anything, but if you do, you can also use the one [implemented on Vec<getter::CrossMsg>](https://github.com/consensus-shipyard/fendermint/pull/331/files#diff-ef5f1e66a6fde4f0d3bfec4815a10f0aa82faee5fd48a38f07a2bfed464bf720R241)

Reply:
We currently don't, but in case we want to do a verification of some kind, so that we are aware of the existence of this :pray: